src/sentry/dynamic_sampling/rules/utils.py

```diff
@@ -145,15 +131,8 @@ def get_rule_hash(rule: PolymorphicRule) -> int:
 
 
 def get_sampling_value(rule: PolymorphicRule) -> Optional[Tuple[str, float]]:
-    # Gets the sampling value from the rule, based on the type.
-    if "samplingValue" in rule:
-        sampling_value = rule["samplingValue"]  # type:ignore
-        return sampling_value["type"], float(sampling_value["value"])
-    # This should be removed once V1 is faded out.
-    elif "sampleRate" in rule:
-        return "sampleRate", rule["sampleRate"]  # type:ignore
-
-    return None
+    sampling_value = rule["samplingValue"]
+    return sampling_value["type"], float(sampling_value["value"])
```

Comment:
```suggestion
    return sampling["type"], float(sampling["value"])
```

Reply:
I will create a PR in which I will remove v1 on its own and will apply the change there!